Makefile

```diff
@@ -1,11 +1,12 @@
-run: build
-	docker-compose up
+run:
+	docker-compose up --build
 build:
 	docker build -t exlibris:latest .
 run-local:
-	docker-compose down
-	docker-compose -f docker-compose.local.yml up
+	docker-compose -f docker-compose.local.yml down
+	docker-compose -f docker-compose.local.yml up -d
+	npm run build
 	go run main.go
```

Comment:
Runs the node build for vue and runs the main.go file locally, dependencies in docker